rasa/utils/io.py

```diff
@@ -234,18 +234,6 @@ def create_path(file_path: Text):
         os.makedirs(parent_dir)
 
 
-def zip_folder(folder: Text) -> Text:
```

Comment:
that's also used by `rasa-x` but I think we can delete it nevertheless and refactor that part in rasa-x (would have to look in it a bit more)

Reply:
ok. but maybe we keep it for now, because we first should refactor rasa-x then. will create an issue for that.

Reply:
ok 👍